resources/views/add_car.blade.php

```diff
@@ -31,29 +31,29 @@
         <form action="{{route('cars.store')}}" method="POST" enctype="multipart/form-data" class="px-md-5">
         @csrf
         <div class="form-group mb-3 row">
-            <label for="" class="form-label col-md-2 fw-bold text-md-end">Car Title:</label>
+            <label for="" class="form-label col-md-2 fw-bold text-md-end">{{ __('cars.carTitle')}}</label>
             <div class="col-md-10">
-              <input type="text" placeholder="BMW" class="form-control py-2" name="carTitle" value="{{old('carTitle')}}"/>
+              <input type="text" placeholder="{{ __('cars.placeholderCarTitle')}}" class="form-control py-2" name="carTitle" value="{{old('carTitle')}}"/>
               @error('carTitle')
-                <div class="alert alert-warning">{{$message}}</div>
+                <div class="alert alert-warning">{{__('cars.titleRequired')}}</div>
```

Comment:
there is a better approach that can be done here
so what if another error happens for example

if the validation rule of decimal fails, the message will not describe what happen

Reply:
you can just modify the file in lang/ar/validation.php

Reply:
Sorry, I didn't see your comment @Nora-Lootfy  but I modified and pushed it within session_15, and i will be blessed if you reviewed it I made commenting to the custom errors that exist in the lang/ar/cars.php and modify the code in lang/ar/validation.php as i added attributes